packages/tasks/src/tasks/zero-shot-object-detection/about.md

```diff
@@ -0,0 +1,15 @@
+## Use Cases
```

Comment:
I think if you're not contributing content to this page you can remove the file itself since placeholder pretty much says the same thing